### ISSUE TITLE
pretrained models for efficientnet_lite0

### ISSUE THREAD

**Issue by xingyueye** [rwightman/gen-efficientnet-pytorch#35]
@rwightman hi, thanks for your work firstly, it's a great repo. 
I want to get the pretrained model for gen-efficientent_lite0, but the url is 'none'. I found a link ''tf_efficientnet_lite0'', is it converted from tensorflow checkpoint? And, what does ' v0.1-weights' means? Can I download it directly and use it normally？

**Comment by rwightman** [comment]
@xingyueye URL is 'none' means there are no weights trained for that model, and yes, you guessed correctly, any model here with `tf_` prefix was converted from an official Tensorflow checkpoint, all of those models require SAME padding support which adds a bit of overhead (not much). The same versions of the model without the tf_ use usuall pytorch padding calculation which is always symmetric. Not all of them have been trained by me or someone else though. My versions of efficientnet are unique in that they allow both the tensorflow compatible padding mode and pytorch mode with runtime argument.

v0.1-weights is the name of the release that I store the binary weight files under, it's convenient place to store weight files with good speed. It's in a different repository though that I share with this... see at https://github.com/rwightman/pytorch-image-models/releases/tag/v0.1-weights

**Comment by xingyueye** [comment]
Thanks a lot, it's helpful
